examples/complete/main.tf

```diff
@@ -16,30 +16,32 @@ module "vpc" {
 module "new_security_group" {
   source = "../.."
 
-  vpc_id = module.vpc.vpc_id
+  vpc_id              = module.vpc.vpc_id
+  open_egress_enabled = true
```

Comment:
I think I would prefer "allow_all_egress" as allow/deny is more standard firewall nomenclature.